Source/WebCore/platform/PlatformSpeechSynthesizer.h

```diff
@@ -72,7 +72,7 @@ class WEBCORE_EXPORT PlatformSpeechSynthesizer : public RefCounted<PlatformSpeec
     // Seems wasteful. Would be nice to find a better way.
     WEBCORE_EXPORT virtual ~PlatformSpeechSynthesizer();
 
-    const Vector<RefPtr<PlatformSpeechSynthesisVoice>>& voiceList() const;
+    virtual const Vector<RefPtr<PlatformSpeechSynthesisVoice>>& voiceList() const;
```

Comment:
Why this?

Reply:
leftovers from previous version, will remove this part

Reply:
fixed